backend/inspirehep/records/api/mixins.py

```diff
@@ -381,7 +390,50 @@ def update(self, data, disable_relations_update=False, *args, **kwargs):
             self.update_conference_paper_and_proccedings()
         else:
             LOGGER.info(
-                "Record conference papaers and proccedings update disabled",
+                "Record conference papers and proceedings update disabled",
                 recid=self.get("control_number"),
                 uuid=str(self.id),
             )
+
+    def get_updated_referenced_conference(self):
```

Comment:
This name is not very intuitive... I would put something like `get_newest_linked_conferences_uuid`. Adding a docstring would also help a lot IMO